site/_config.yml

```diff
@@ -25,9 +25,10 @@ pulsar_repo: https://github.com/apache/incubator-pulsar/tree/master
 baseurl: /
 destination: ../generated-site/content
 
-preview_version_id: 20180426.125800-32
-current_version: 1.22.0-incubating
+preview_version_id: 20180330.125917-6
+current_version: "2.0.0"
```

Comment:
@lucperkins  I think some of my previous comments got buried up earlier. 

The moment we change current version, it will change the download links as well. This is done once the release is finally approved.

I think we should leave 1.22 here, and merge this docs changes. We'll change the current version as part of release process.